odata-json-format/meta.yaml

```diff
@@ -0,0 +1,8 @@
+pagetitle: OData JSON Format Version 4.02
+subtitle: Committee Specification Draft 01
+filename: odata-json-format-v4.02-csd01
```

Comment:
filename deviates from pattern used in other meta.yaml, eg for odata-protocol

Reply:
This is the OASIS filename for publication, see Paul's mail https://www.oasis-open.org/apps/org/workgroup/odata/email/archives/202307/msg00018.html:

> When the TC first votes [2] to publish these Work Products in the OASIS Library, we expect them to be published at:
> - https://docs.oasis-open.org/odata/odata/v4.02/csd01/odata-v4.02-csd01-part1-protocol.html
> - https://docs.oasis-open.org/odata/odata/v4.02/csd01/odata-v4.02-csd01-part2-url-conventions.html
> *** (Note that individual directories for Part 1 and Part 2 are no longer used for this multi-part specification ***)
> - https://docs.oasis-open.org/odata/odata-csdl-json/v4.02/csd01/odata-csdl-json-v4.02-csd01.html
> - https://docs.oasis-open.org/odata/odata-csdl-xml/v4.02/csd01/odata-csdl-xml-v4.02-csd01.html
> - https://docs.oasis-open.org/odata/odata-json-format/v4.02/csd01/odata-json-format-v4.02-csd01.html

Protocol and URL Conventions together form a multi-part document, so they use a slightly different filename pattern.